color_refinement.py

```diff
@@ -98,40 +107,47 @@ def get_number_isomorphisms(g: "Graph", h: "Graph", count: bool) -> int:
     return count_isomorphism(g, h, coloring, count)
 
 
-def is_isomorphisms(g: "Graph", h: "Graph") -> bool:
+def is_isomorphisms(g: Graph, h: Graph) -> bool:
+    """
+    Returns whether the two graphs are isomorphic
+
+    The algorithm of `get_number_isomorphisms` with count set to `False` is used to determine the number of
+    isomorphisms. When the number of isomorphisms is 0, graphs are not isomorphic. Otherwise, the graphs are isomorphic.
```

Comment:
Volgens mij hoord deze zin niet hier? Want de functie is `is_isomorphisms` is de functie waar het bij staat, maar het gaat over `get_number_isomorphisms`

Reply:
Is misschien duidelijk als we `Uses the algorithm ...`